diff -up ./lib/rapidxml/rapidxml_iterators.hpp.orig ./lib/rapidxml/rapidxml_iterators.hpp
--- ./lib/rapidxml/rapidxml_iterators.hpp.orig	2018-02-05 09:22:58.000000000 +0100
+++ ./lib/rapidxml/rapidxml_iterators.hpp	2018-03-15 02:53:30.000985600 +0100
@@ -7,7 +7,7 @@
 //! \file rapidxml_iterators.hpp This file contains rapidxml iterators
 
 #include <iterator>
-#include "rapidxml.hpp"
+#include <rapidxml.h>
 
 namespace rapidxml
 {
diff -up ./lib/rapidxml/rapidxml_print.hpp.orig ./lib/rapidxml/rapidxml_print.hpp
--- ./lib/rapidxml/rapidxml_print.hpp.orig	2018-02-05 09:22:58.000000000 +0100
+++ ./lib/rapidxml/rapidxml_print.hpp	2018-03-15 02:53:29.996985693 +0100
@@ -6,7 +6,7 @@
 // Revision $DateTime: 2009/05/13 01:46:17 $
 //! \file rapidxml_print.hpp This file contains rapidxml printer implementation
 
-#include "rapidxml.hpp"
+#include <rapidxml.h>
 
 // Only include streams if not disabled
 #ifndef RAPIDXML_NO_STREAMS
diff -up ./lib/rapidxml/rapidxml_utils.hpp.orig ./lib/rapidxml/rapidxml_utils.hpp
--- ./lib/rapidxml/rapidxml_utils.hpp.orig	2018-02-05 09:22:58.000000000 +0100
+++ ./lib/rapidxml/rapidxml_utils.hpp	2018-03-15 02:53:30.003985529 +0100
@@ -7,7 +7,7 @@
 //! \file rapidxml_utils.hpp This file contains high-level rapidxml utilities that can be useful
 //! in certain simple scenarios. They should probably not be used if maximizing performance is the main objective.
 
-#include "rapidxml.hpp"
+#include <rapidxml.h>
 #include <vector>
 #include <string>
 #include <fstream>
diff -up ./src/PVRIptvData.cpp.orig ./src/PVRIptvData.cpp
--- ./src/PVRIptvData.cpp.orig	2018-02-05 09:22:58.000000000 +0100
+++ ./src/PVRIptvData.cpp	2018-03-15 02:54:00.617229220 +0100
@@ -30,7 +30,7 @@
 #include <string>
 
 #include "zlib.h"
-#include "rapidxml/rapidxml.hpp"
+#include <rapidxml.h>
 #include "PVRIptvData.h"
 #include "p8-platform/util/StringUtils.h"
 #include "client.h"
